a10_octavia/controller/worker/tasks/handler_server.py

```diff
@@ -22,87 +22,95 @@
 
 
 class MemberDelete(BaseVThunderTask):
     """ Task to delete member """
 
     def execute(self, member, vthunder, pool):
         """ Execute delete member """
+        c = self.client_factory(vthunder)
         try:
-            c = self.client_factory(vthunder)
-            c.slb.service_group.member.delete(pool.id, member.id, member.protocol_port)
-            LOG.info("Member dissociated from pool successfully.")
+            c.slb.service_group.member.delete(
+                pool.id, member.id, member.protocol_port)
+            LOG.debug("Member %s is dissociated from pool %s successfully.", member.id, pool.id)
             c.slb.server.delete(member.id)
-            LOG.info("Member deleted successfully.")
+            LOG.debug("Member %s is deleted successfully.", member.id)
         except Exception as e:
-            LOG.error(str(e))
-            LOG.info("Error occurred")
+            LOG.warning("Failed to delete member: %s", str(e))
 
 
 class MemberUpdate(BaseVThunderTask):
     """ Task to update member """
 
     def execute(self, member, vthunder, pool):
-        """Execute create member for an amphora."""
-        conn_limit = CONF.server.conn_limit
-        conn_resume = CONF.server.conn_resume
+        """Execute update member """
         server_args = self.meta(member, 'server', {})
+        server_args['conn-limit'] = CONF.server.conn_limit
+        server_args['conn-resume'] = CONF.server.conn_resume
+        server_args = {'server': server_args}
+        server_temp = {}
+        server_temp['template-server'] = CONF.server.template_server
+        if not member.enabled:
+            status = False
+        else:
+            status = True
+        c = self.client_factory(vthunder)
 
         try:
-            c = self.client_factory(vthunder)
-            if not member.enabled:
-                status = False
-            else:
-                status = True
-            server_args['conn-limit'] = conn_limit
-            server_args['conn-resume'] = conn_resume
-            server_args = {'server': server_args}
-            try:
-                conf_templates = CONF.server.template_server
-                server_temp = {}
-                server_temp['template-server'] = conf_templates
-            except:
-                server_temp = None
-                LOG.error("Invalid definition of A10 config in Member section.")
-
             c.slb.server.update(member.id, member.ip_address, status=status,
                                 server_templates=server_temp,
                                 axapi_args=server_args)
-            LOG.info("Member updated successfully.")
+            LOG.debug("Member updated successfully: %s", member.id)
         except Exception as e:
-            LOG.error(str(e))
-            LOG.info("Error occurred")
+            LOG.exception("Failed to update member: %s", str(e))
+            raise
+
+    def revert(self, member, vthunder, pool, *args, **kwargs):
```

Comment:
We don't need to revert when updating. If it fails then the failure is simply raised.